From 0e33000c917458cd1c6d884377d6442d50b14a58 Mon Sep 17 00:00:00 2001
From: Dongwon Kim <dongwon.kim@intel.com>
Date: Wed, 4 Jun 2025 16:06:41 -0700
Subject: [PATCH 50/50] gtk: Abort if there is no connector set for primary
 window

Signed-off-by: Dongwon Kim <dongwon.kim@intel.com>
---
 ui/gtk.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/ui/gtk.c b/ui/gtk.c
index eeb41cb496..3b5b550b64 100644
--- a/ui/gtk.c
+++ b/ui/gtk.c
@@ -1905,14 +1905,18 @@ static void gd_connectors_init(GdkDisplay *dpy, GtkDisplayState *s)
         }
         if (first_vc) {
             vc->window = s->window;
-            first_vc = FALSE;
         }
 
         vc->label = g_strdup(conn->value);
         monitor_num = gd_monitor_lookup(dpy, vc->label);
         if (monitor_num >= 0) {
             gd_window_show_on_monitor(dpy, vc, monitor_num);
+        } else if (first_vc) {
+            fprintf(stderr, "gtk: no monitor found for '%s' so terminating..\n",
+                    vc->label);
+            abort();
         }
+        first_vc = FALSE;
     }
 
     mon_reset_timer = timer_new_ms(QEMU_CLOCK_REALTIME,
-- 
2.35.3

